### DISCUSSION TITLE
Is there a way to make the settings menu scrollable?

### DISCUSSION THREAD

**Discussion by TalinTheDev** [TheWisker/Shikai#9] [Q&A]
I'm posting this here instead of in issues because I tried adding `overflow: scroll` to the menu however the makes a scrollbar appear even when it is closed and for some reason, I cannot hide it so I'm assuming @TheWisker already tried something out with that.

If it's possible, I'm willing to help implement it, it is just annoying because I would like to change the caret to | on the right but when I go down, I can't exit out of the menu to save it.

Thanks!

**Comment by TheWisker** [comment]
I have tried a few solutions to that already but all of them always lack something: one always shows the scrollbars, another one clips the color choosers, etc. So I do not know exactly what to do. If you are willing to help you are more than welcome, it is one of the trickiest css problems I have ever faced.

**Reply by TalinTheDev** [reply]
Okay i'll try somethings and let you know how it goes!

**Reply by TheWisker** [reply]
Should be fixed by now!
